src/App.js

```diff
@@ -748,15 +742,12 @@ export class App extends React.Component<AppProps, AppState> {
                 && this.state.runningState === 'running') {
             this.interpreter.startRun();
         }
-        if (this.state.settings.theme !== prevState.settings.theme) {
-            if (document.body) {
-                if (this.state.settings.theme === 'default') {
-                    document.body.className = '';
-                } else {
-                    document.body.className = `${this.state.settings.theme}-theme`;
-                }
-            }
+
+        // As long as we have no underlying "default" theme styles, we need to always set this to ensure that the default theme is applied on startup.
```

Comment:
Right place to set this up startup behaviours would be in ComponentDidMount.

Reply:
This results in a short "blip" on refresh, where the default theme appears briefly and then is replaced.  I'll commit what I have so we can pair on it.